frontend/src/components/NewPassword/NewPassword.js

```diff
@@ -0,0 +1,90 @@
+//  global sessionStorage  eslint-disable-line no-undef
+/* eslint no-undef: "error" */
+
+import React, { Component } from 'react';
+import { loginUser } from '../../lib/apiClient';
+import './NewPassword.css';
+
+class NewPassword extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      isPasswordVisible: false,
+      password: '',
+    };
+  }
+
+  toggleIcon = () => {
+    this.setState(prevState => ({
+      isPasswordVisible: !prevState.isPasswordVisible,
+    }));
+  };
+
+  handleChange = (event) => {
+    this.setState({ [event.target.name]: event.target.value });
+  };
+
+  handleSubmit = (event) => {
+    event.preventDefault();
+    loginUser(this.state);
```

Comment:
We should just remove this, since it won't work as is and won't be used when this is hooked up to the backend.